modules/www/main.tf

```diff
@@ -407,28 +422,51 @@ resource "aws_ecs_task_definition" "www_task_definition" {
        "privileged": false,
        "name": "nerves_hub_www",
        "environment": [
-         {
-           "name": "ENVIRONMENT",
-           "value": "${terraform.workspace}"
-         },
-         {
-           "name": "APP_NAME",
-           "value": "nerves_hub_www"
-         }
+         ${local.ecs_shared_env_vars}
        ],
+       "volumesFrom": [],
+       "mountPoints": [],
        "logConfiguration": {
```

Comment:
I think this is where we pass in the logConfiguration and we can remove this section 🤔